internal/storage/elasticsearch/config/config.go

```diff
@@ -234,23 +234,31 @@ func NewClient(c *Configuration, logger *zap.Logger, metricsFactory metrics.Fact
 		}).
 		After(func(id int64, requests []elastic.BulkableRequest, response *elastic.BulkResponse, err error) {
 			start, ok := m.Load(id)
-			if !ok {
-				return
+			if ok {
+				m.Delete(id)
+			} else {
+				start = time.Now()
 			}
-			m.Delete(id)
 
 			// log individual errors, note that err might be false and these errors still present
-			if response != nil && response.Errors {
-				for _, it := range response.Items {
-					for key, val := range it {
-						if val.Error != nil {
-							logger.Error("Elasticsearch part of bulk request failed", zap.String("map-key", key),
-								zap.Reflect("response", val))
+			if response != nil {
```

Comment:
this code appears nearly identical to what it was before, why does it need to change?